lib/Db/ItemMapper.php

```diff
@@ -64,7 +64,7 @@ private function buildStatusQueryPart($showAll, $type = null)
         if (isset($type) && $type === FeedType::STARRED) {
             $sql = 'AND `items`.`starred` = ';
             $sql .= $this->db->quote(true, IQueryBuilder::PARAM_BOOL) . ' ';
-        } elseif (!$showAll) {
+        } elseif (!$showAll || $type == FeedType::UNREAD) {
```

Comment:
```
    line   67: == - is discouraged

App is not compliant
```

Reply:
I have fixed that to use the strict equal comparison.